cardano-node/README.md

```diff
@@ -43,19 +143,22 @@ set -g default-terminal "tmux-256color"
 
     `./scripts/shelley-testnet.sh`
 
-The window of the terminal will be split into four panes showing the three
-nodes running and a shell to enter commands for transaction submission, e.g.
+## Submit a tx to the testnet
+
+Following the instructions above the window of the terminal will be split into four panes.
```

Comment:
Grammar nitpicking: "Following the instructions above, the window of the terminal will be split into four panes."